engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/sink/KafkaAvroSink.scala

```diff
@@ -22,10 +22,11 @@ class KafkaAvroSink(topic: String, output: LazyParameter[Any], kafkaConfig: Kafk
       .map(flinkNodeContext.lazyParameterHelper.lazyMapFunction(output))
       .map(ctx => ctx.value match {
           case data: java.util.Map[String, Any] => BestEffortAvroEncoder.encodeRecordOrError(data, schema)
-          case _: GenericRecord => ctx.value
+          case record: GenericRecord => record
           case value => {
+            //TODO: We should better handle this situation by using EspExceptionHandler
             logger.error("Invalid output type error.", ctx)
-            throw InvalidSinkOutput(s"Invalid output type ${value.getClass}. Excepted java.util.Map or GenericRecord.")
+            value
```

Comment:
I think value is not good choice as it will blow up in a moment? If we want to continue processing all we can do is log error and drop record?